README.md

```diff
@@ -2,89 +2,194 @@
 
 OUSD is a new kind of stablecoin that passively accrues yield while you are holding it.
 
-## Development
+## Requirements
+- Node Version
+  - `^8.11.2 >= node <=^14.0.0`
+  - Recommended `^14.0.0`
+- Web3 Wallet
+  - Recommended [Metamask](https://metamask.io/) 
 
-Ethereum tests and local Ethereum EVM are managed by Hardhat.
+---
 
-A variety of Hardhat tasks are available to interact with the contracts. Additional information can be found by running `npx hardhat` from the `contracts/` directory.
+## Installation
+```bash
+# Clone the origin-dollar project
+git clone git@github.com:OriginProtocol/origin-dollar.git
+```  
 
+---
 
-### Running local node using Hardhat EVM
+## Description
 
-`cd contracts`
+The `origin-dollar` project is a mono repo that houses both the `smart contracts` and `dApp` codebases. In order to run this project locally, you will need to run both the `node` and the `dapp` in separate processes or terminals. 
 
-`yarn install && yarn run node`
 
-Start the DApp in another window:
+### Eth Node
+The `smart contracts` and all of their associated code is located in the `<project-root>/contracts` directory.Ethereum tests and local Ethereum EVM are managed by [Hardhat](https://hardhat.org/).
 
-`cd dapp`
+A variety of Hardhat [tasks](https://hardhat.org/guides/create-task.html) are available to interact with the contracts. Additional information can be found by running `npx hardhat` from the `contracts/` directory.
+<br/><br/>
 
-`yarn install && yarn start`
+### dApp(Decentralized Application)
+The `dApp` and it's associated code is located in the `<project-root>/contracts` directory.
+<br/><br/>
 
-Connect MetaMask to `http://localhost:8545`.
+---
+## Developing Locally
 
-When freshly starting a node it is usually necessary to also reset Metamask Account being used:
-- Click on Account top right icon -> settings -> advanced -> Reset Account
+You have two options for running the Ethereum node locally via hardhat.
+- Standalone mode - A private blockchain with a clean slate
+- Forked mode - A forked version of mainnet at a particular block height
+
+The dApp will be started in development mode bby default and will assume the underlying blockchain state it is connected to.
```

Comment:
nit: typo: `bby`

Reply:
Docs might need also slight adjustement. For standalone mode the dapp should be started using `yarn run start` and for forked mode using `yarn run start:fork`

Reply:
Thanks for the 👀  - done